demo/js/components/accessibility-demo.js

```diff
@@ -103,6 +123,117 @@ export default class App extends React.Component {
               />
             </VictoryChart>
           </div>
+
+          {/** AREA */}
+          <div style={chartContainerStyle}>
+            <h3 style={chartHeadingStyle}> Area </h3>
+            <VictoryChart domainPadding={{ y: 10 }}>
+              <VictoryStack>
+                <VictoryArea
+                  data={accessibilityAreaData.a}
+                  dataComponent={
+                    <Area
+                      ariaLabel={({ data }) => `area chart stack ${data[0]._stack}`}
+                      tabIndex={20}
+                    />
+                  }
+                />
+                <VictoryArea
+                  data={accessibilityAreaData.b}
+                  dataComponent={
+                    <Area
+                      ariaLabel={({ data }) => `area chart stack ${data[0]._stack}`}
+                      tabIndex={20.1}
+                    />
+                  }
+                />
+                <VictoryArea
+                  data={accessibilityAreaData.c}
+                  dataComponent={
+                    <Area
+                      ariaLabel={({ data }) => `area chart stack ${data[0]._stack}`}
+                      tabIndex={20.2}
+                    />
+                  }
+                />
+                <VictoryArea
+                  data={accessibilityAreaData.d}
+                  dataComponent={
+                    <Area
+                      ariaLabel={({ data }) => `area chart stack ${data[0]._stack}`}
+                      tabIndex={20.3}
+                    />
+                  }
+                />
+              </VictoryStack>
+            </VictoryChart>
+          </div>
+
+          {/** LINE */}
+          <div style={chartContainerStyle}>
+            <h3 style={chartHeadingStyle}> Line </h3>
+            <VictoryChart domain={{ x: [0, 6], y: [0, 7] }}>
+              <VictoryLine
+                data={accessibilityLineDemo}
+                labels={({ datum }) => datum.y}
+                // labelComponent={<VictoryLabel datum />}
```

Comment:
maybe going about this wrong or missing a feature that already exists to add aria-labels to chart labels

Reply:
I'm not sure if I'm answering the right question here, but, `VictoryLabel` should already be getting access to datum from `VictoryLine`. `VictoryLabel` should definitely get the new `ariaLabel` prop, though.

Reply:
You could use it like this: 

```
<VictoryLabel verticalAnchor={({ datum }) => datum.y > 0 ? "start" : "end"}  />
```

Reply:
I dont know what the issue was yesterday, its very possible I was making changes in one demo file and running another 😳  all good today and working as expected. Thank you!